src/App.js

```diff
@@ -3,16 +3,24 @@ import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
 import logo from './logo.svg';
 import './App.css';
-
+import LoginWithAmazon from './LoginWithAmazon.js'
 import ChatWindow from './ChatWindow.js'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
```

Comment:
I think you should initialize access_token with an empty string.

Reply:
Also, we might want to include expiry time in here. Put access token and expiry time in an object instead of adding them as two independent fields in the state.